rpm/build/SOURCES/jenkins.sysconfig.in

```diff
@@ -2,7 +2,7 @@
 ## Description: @@SUMMARY@@
 ## Type:        string
 ## Default:     "~~HOME~~"
-## ServiceRestart: jenkins
+## ServiceRestart: @@ARTIFACTNAME@@
```

Comment:
If you do this then service start Jenkins does not start jenkins but `@@ARTIFACTNAME@@` which if it is not jenkins becomes unexpected.